src/components/Loading/tests/Loading.test.tsx

```diff
@@ -68,10 +67,6 @@ describe('<Loading />', () => {
 
 function mountWithAppBridge(element: React.ReactElement<any>) {
```

Comment:
What do you think about removing this function?

Reply:
AppBridge stuff is deprecated and will be cleared away in v5, I'm tempted to leave this as-is to keep diff noise down and it'll all be washed away later.